build.sbt

```diff
@@ -102,15 +103,28 @@ lazy val codegen = project
   .settings(name := "caliban-codegen")
   .settings(commonSettings)
   .settings(
-    sbtPlugin := true,
     crossScalaVersions := Seq("2.12.10"),
```

Comment:
This line can be remove (only needed for the plugin).

Reply:
You missed this one (I meant `crossScalaVersions` should be removed, this was to force the sbt plugin to be built on 2.12 only but a regular module can be cross-published.

Reply:
I missed that one. Updated